lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -36,9 +36,46 @@ function collectImageElementInfo() {
       clientHeight: element.clientHeight,
       naturalWidth: element.naturalWidth,
       naturalHeight: element.naturalHeight,
+      isCss: false,
       isPicture: element.parentElement.tagName === 'PICTURE',
     };
   });
+
+  // Chrome normalizes background image style from getComputedStyle to be an absolute URL in quotes.
+  // Only match basic background-image: url("http://host/image.jpeg") declarations
+  const CSS_URL_REGEX = /^url\("([^"]+)"\)$/;
+  // Only find images that aren't specifically scaled
+  const CSS_SIZE_REGEX = /(auto|contain|cover)/;
+  const cssImages = [...document.querySelectorAll('*')].reduce((images, element) => {
```

Comment:
Collecting all the dom nodes on the page + running `gCS` on them....wowza. How heavy is that?